dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -569,6 +568,11 @@ private void updateInfo(Workflow oldWorkflow, Workflow newWorkflow) {
             oldWorkflow.setTopicSelection(newWorkflow.getTopicSelection());
         }
 
+        // If the user chose an AI topic, it means that they have reviewed and approved it
```

Comment:
Ditto, for example if the user meant just to update the defaultWorkflowPath, would the approval be set to true accidentally?